bundles/org.openhab.binding.boschshc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -422,6 +443,7 @@
 
 		<label>Universal Switch II</label>
 		<description>Universally configurable switch with four buttons.</description>
+		<semantic-equipment-tag>WallSwitch</semantic-equipment-tag>
```

Comment:
Universal Switch II (and I assume also Universal Switch I, which is not sold anymore) are portable switches, but can also be mounted on walls by means of a mounting plate: https://www.bosch-smarthome.com/uk/en/products/control-elements-and-accessories/universal-switch/

Is there a semantic tag for portable switches / remote controls? IMHO it would be more appropriate than `WallSwitch`. And if not, is there a different tag for switches mounted on top of walls as opposed to switches mounted inside walls? Or is this not important?

Is there an overview page where all tags are listed?

Reply:
> Or is this not important?

IMHO not important.

> overview page where all tags are listed

See this https://github.com/openhab/openhab-core/blob/ad7612dba7963ef93c140bebb743aa3aab429928/bundles/org.openhab.core.semantics/model/SemanticTags.csv

Reply:
Thanks for the link ❤️  `ControlDevice` sounds good to me 👍